api/v1/synthesizer.go

```diff
@@ -33,19 +33,19 @@ type SynthesizerSpec struct {
 	// Timeout for each execution of the synthesizer command.
 	//
 	// +kubebuilder:default="10s"
-	ExecTimeout metav1.Duration `json:"execTimeout,omitempty"`
+	ExecTimeout *metav1.Duration `json:"execTimeout,omitempty"`
```

Comment:
Imagine how many times I facepalmed upon realizing that non-ptr durations don't support defaulting

Reply:
Outrageous!!!